test/faker/address/es_test.exs

```diff
@@ -0,0 +1,88 @@
+defmodule Faker.Address.EsTest do
+  use ExUnit.Case, async: true
+
+  test "building_number/0" do
+    assert is_binary(Faker.Address.Es.building_number)
+    assert String.length(Faker.Address.Es.building_number) in 1..5
```

Comment:
this kind of tests make sense, but should be like the one in `color_test.exs` for example. While is_binary should go to doctests.